backend/tarsy/main.py

```
@@ -441,11 +444,20 @@
         return {"alert_id": alert_id, "session_id": None}
 
 @app.websocket("/ws/dashboard/{user_id}")
-async def dashboard_websocket_endpoint(websocket: WebSocket, user_id: str):
+async def dashboard_websocket_endpoint(
+    websocket: WebSocket, 
+    user_id: str,
+    jwt_payload: Optional[Dict[str, Any]] = Depends(verify_jwt_token_websocket)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)